concepticondata/concepticon.tsv

```diff
@@ -3148,3 +3148,33 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 3152	MUNTJACS	Animals	Also known as barking deer, a kind of small deer of the genus Muntiacus.	Person/Thing	
 3153	LORD	Social and political relations	A man of noble rank or high office; a nobleman.	Person/Thing	
 3154	WILD BOAR	Animals	Also known as wild pig, an undomestic four-legged animal (Sus scrofa) living in the wild. 	Person/Thing	
+3155	DIBBLE STICK	Agriculture and vegetation	A piece of wood with a point used for making holes in the ground for seeds, seedlings or small bulbs.	Person/Thing	
+3156	GEBANG PALM	Agriculture and vegetation	Tree of the Corypha Utan species.	Person/Thing	
+3157	CIVET	Animals	A carnivorous mammal of the Viverridae family, found across South East Asia.	Person/Thing	
+3158	RICE EAR BUG	Animals	An insect of the species Leptocorisa oratoria, which gives of an unpleasant smell when disturbed and feeds on rice plants, destroying the crops.	Person/Thing	
+3159	BURDEN STICK	Basic actions and technology	A long piece of wood used to carry a burden.	Person/Thing	
+3160	SLAP	Basic actions and technology	Hit with the open palm of the hand	Action/Process	
+3161	CHEW BETEL	Food and drink	To consume a mixture of areca nut and betel leaf.	Action/Process	
+3162	CROSS-COUSIN	Kinship	The child of a parent’s opposite-sex sibling, for example the father’s sister’s son.	Person/Thing	
+3163	SQUAT	Motion	Have bent knees with the body weight is on the feet.	Action/Process	
+3164	ONE HUNDRED AND TWENTY THREE	Quantity	The cardinal number represented in Roman numerals as CXXIII, and in Arabic numerals as 123.	Other	
+3165	TWENTY THREE	Quantity	The cardinal number represented in Roman numerals as XXIII, and in Arabic numerals as 23.	Other	
+3166	TWENTY TWO	Quantity	The cardinal number represented in Roman numerals as XXII, and in Arabic numerals as 22.	Other	
+3167	SERMON	Religion and belief	A religious lecture or speech, addressing issues of religion or morality.	Person/Thing	
+3168	DOWRY	Social and political relations	Property, gifts or money transfered from the parents of a daughter to her new family at marriage.	Person/Thing	
+3169	TREATY	Social and political relations	Formal agreement between nations, states or other international bodies.	Person/Thing	
+3170	BATHE SOMEONE	The body	Clean someone else with water.	Action/Process	
```

Comment:
BATHE (SOMEONE) is a better way to gloss this.

Reply:
I thought I had that. Will fix.